charts/emissary-ingress/values.yaml.in

```diff
@@ -254,6 +254,14 @@ restartPolicy:
 
 terminationGracePeriodSeconds:
 
+waitForApiext:
+  enabled: false
```

Comment:
I think we want this enabled by default right? Then the user can opt out if they do not want the init containers.

Reply:
@AliceProxy - thoughts?